<launch>
	<!--env name="ROSCONSOLE_CONFIG_FILE" value="$(find next_best_view)/log/rosconsole.config" /-->
	<include file="$(find mild_navigation)/launch/simulation_floor.launch" />
	<include file="$(find world_model)/launch/world_model.launch" />
	<include file="$(find recognition_for_grasping)/launch/recognition_manager.launch" />
	<include file="$(find ptu_driver)/launch/ptu_left_mock.launch" />
	<include file="$(find next_best_view)/launch/RobotModelServiceSim.launch" />
	<node pkg="next_best_view" type="next_best_view_parameters_test" name="nbv" output="screen">

		<param name="output_path" value="$(find next_best_view)/rsc/" />
	
		<rosparam file="$(find next_best_view)/rsc/next_best_view_settings_sim.yaml"/>
          	<rosparam file="$(find next_best_view)/rsc/VisualizationConfig.yaml"/>
		<rosparam file="$(find next_best_view)/rsc/TopicsConfig.yaml"/>
                <rosparam file="$(find next_best_view)/rsc/costmap_params.yaml" command="load"/>
                <rosparam file="$(find mild_navigation)/config/simulation/global_costmap_params.yaml" command="load"/>
                <param name="mCropBoxListFilePath" value="$(find next_best_view)/rsc/CropBoxListHighResolution.xml"/>
                <param name="mMapsImagePath" value="$(find next_best_view)/rsc"/>

		<param name="positions/test" value="true"/>
		<rosparam param="positions/element_types">["CeylonTea", "Smacks", "CoffeeFilters2"]</rosparam>
		<rosparam param="positions/element_orientation">[-90.0 , -90.0, 0.0]</rosparam>
		
		<param name="configurable_point_clouds/test" value="false"/>
		<param name="side_objects/test" value="false"/>
	</node>
</launch>
